components/02-molecules/banner/grand-hero/_yds-grand-hero.scss

```diff
@@ -0,0 +1,108 @@
+@use '../../../00-tokens/tokens';
+@use '../../../01-atoms/atoms';
+
+$break-grand-hero-banner: tokens.$break-m;
+
+.grand-hero-banner {
+  --site-header-height: 77px;
+
+  @include tokens.spacing-page-section($flush-top: true, $flush-bottom: true);
+
+  position: relative;
+  overflow: hidden;
+  display: grid;
+  align-items: end;
+
+  &[data-grand-hero-overlay-variation='contained'] {
+    @media (min-width: $break-grand-hero-banner) {
+      justify-content: start;
+    }
+  }
+
+  @media (min-width: $break-grand-hero-banner) {
+    align-items: center;
+  }
+
+  &[data-grand-hero-size='reduced'] {
+    min-height: 28rem;
+  }
+
+  &[data-grand-hero-size='full'] {
+    min-height: calc(95vh - var(--site-header-height));
+  }
+}
+
+.grand-hero-banner__image {
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+
+  img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+}
+
+.grand-hero-banner__content {
+  position: relative;
+  padding: var(--size-spacing-5) var(--size-spacing-7);
+  background: transparent;
+
+  [data-grand-hero-overlay-variation='contained'] & {
+    margin-bottom: var(--size-spacing-6);
+
+    @media (min-width: $break-grand-hero-banner) {
+      margin: var(--size-spacing-site-gutter);
+    }
+  }
+
+  [data-grand-hero-overlay-variation='full'] & {
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    background-color: var(--color-background);
+    opacity: 0.6;
+  }
+}
+
+.grand-hero-banner__content-inner {
+  position: relative;
+  display: grid;
+  gap: var(--size-spacing-3);
+
+  @media (min-width: $break-grand-hero-banner) {
+    max-width: var(--size-component-layout-width-content);
+  }
+
+  [data-grand-hero-overlay-variation='full'] & {
+    padding-left: var(--size-spacing-8);
```

Comment:
This should be `padding-left: var(--size-spacing-site-gutter);` because the "site gutter" is the exact variable that needs to be added. (And it changes at breakpoints, so we need to use it so that it's in sync with other components across all screen sizes.)